concept/migration/data-validation.feature

```diff
@@ -8,8 +8,8 @@ Feature: Data validation
   Background:
     Given typedb starts
     Given connection opens with default authentication
-    Given connection has been opened
-    Given connection does not have any database
+    Given connection is open: true
```

Comment:
Can choose between `is open: true / is open: false` and `is open / is not open`. I'm not against any of the options and use them in the mix.

Reply:
These changes will affect the server. I'll create a server PR once I get approval on this one, and I'll update the server's ref in the respective PR as well.

Reply:
Actually, here we go https://github.com/typedb/typedb/pull/7198/files